e2eTests/e2eTerraformAuthPermissionMismatch_test.go

```diff
@@ -0,0 +1,87 @@
+//     MIT License
+//
+//     Copyright (c) Microsoft Corporation.
+//
+//     Permission is hereby granted, free of charge, to any person obtaining a copy
+//     of this software and associated documentation files (the "Software"), to deal
+//     in the Software without restriction, including without limitation the rights
+//     to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//     copies of the Software, and to permit persons to whom the Software is
+//     furnished to do so, subject to the following conditions:
+//
+//     The above copyright notice and this permission notice shall be included in all
+//     copies or substantial portions of the Software.
+//
+//     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//     IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//     FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//     AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//     LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//     SOFTWARE
+
+package e2etests
+
+import (
+	"context"
+	"os"
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/Azure/mpf/pkg/infrastructure/authorizationCheckers/terraform"
+	resourceGroupManager "github.com/Azure/mpf/pkg/infrastructure/resourceGroupManager"
```

Comment:
You do not need to alias this. The pkg name is the same as alias.

Reply:
on saving the file go fmt is automatically adding the alias for readability. One of the reasons is there are variables being used in the file with same name as package spRoleAssignmentManager. Issue https://github.com/Azure/mpf/issues/43 has been created to address this issue across all e2e tests